### PR TITLE
Expose battery state api

### PR DESCRIPTION
**Pull request by colombod**
expose battery state api
events for battery state changed
events for battery warning issued
rename pitopmodule to pitop4board

### CODE REVIEW

File: src/SampleApp/Program.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ static async Task Main(string[] args)
             Console.WriteLine("ready");
 
             Console.WriteLine(@"Select one of the options
+0 module test
```
Comment (m-roberts) [inline src/SampleApp/Program.cs:21]: Module or board?

File: src/SampleApp/Program.cs [hunk 2]
```diff
@@ -51,11 +55,22 @@ static async Task Main(string[] args)
             Console.WriteLine("done");
         }
 
+        private static Task TestModule()
```
Comment (m-roberts) [inline src/SampleApp/Program.cs:58]: Module or board?

File: src/PiTop.Camera.App/Program.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            using var module = PiTopModule.Instance
+            using var module = PiTop4Board.Instance
```
Comment (m-roberts) [inline src/PiTop.Camera.App/Program.cs:10]: Module or board?

**Review by m-roberts — APPROVED** [review]
Looks good to me - except you changed it from a module to a board, without keeping the naming consistent. But this isn't critical

File: src/SampleApp/Program.cs (hunk 1 above)
Comment (colombod) [reply src/SampleApp/Program.cs:21]: done

File: src/PiTop.Camera.App/Program.cs (hunk 3 above)
Comment (colombod) [reply src/PiTop.Camera.App/Program.cs:10]: done

File: src/SampleApp/Program.cs (hunk 2 above)
Comment (colombod) [reply src/SampleApp/Program.cs]: done

**Pull request merged**